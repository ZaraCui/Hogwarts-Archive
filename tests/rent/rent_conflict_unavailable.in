ADD COLLECTION tests/data/books_small.csv
ADD STUDENT Harry Potter
ADD STUDENT Draco Malfoy
RENT 100000 1001
RENT 100001 1001
EXIT

