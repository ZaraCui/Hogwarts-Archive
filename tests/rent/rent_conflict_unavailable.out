user: 12 spellbooks successfully added.

user: Success.

user: Success.

user: Success.

user: Spellbook is currently unavailable.

user: Ending Archive process.
